scripts/extract/check_new_date.py

```diff
@@ -52,13 +51,31 @@ def prepare_df_to_append(new_sitting_dates, version=2):
 ### Main Run ###
 
 
-def process(seeds_date_filepath):
-    df = pd.read_csv(seeds_date_filepath)
+def process():
+
+    # read from gbq instead of from filepath
+
+    df = pandas_gbq.read_gbq(query_or_table = """
+                                    SELECT *
+                                    FROM `singapore-parliament-speeches.raw.dates`
+                                    """)
```

Comment:
````suggestion
    df = pandas_gbq.read_gbq(query_or_table = """
                                    SELECT Sitting_Date
                                    FROM `singapore-parliament-speeches.raw.dates`
                                    """)
````

to reduce bytes read (more cost-effective), select the specific column which needs to be read.